stable_baselines3/common/base_class.py

```diff
@@ -325,6 +326,23 @@ def _excluded_save_params(self) -> List[str]:
             "_custom_logger",
         ]
 
+    def _get_policy_from_name(self, policy_name: str) -> Type[BasePolicy]:
+        """
+        Get a policy class from its name representation.
+
+        The goal here is to standardize policy naming, e.g.
+        all algorithms can call upon "MlpPolicy" or "CnnPolicy",
+        and they receive respective policies that work for them.
+
+        :return:
+            A policy class (type)
+        """
+
+        if policy_name in self.policy_aliases:
+            return self.policy_aliases[policy_name]
+        else:
+            raise ValueError("Policy %s unknown" % policy_name)
```

Comment:
```suggestion
            raise ValueError(f"Policy {policy_name} unknown")
```

Please use f-strings